.jenkins/build-sanity.groovy

```diff
@@ -35,28 +35,55 @@ node('cico-workspace') {
 			}
 			firstAttempt = false
 			cico = sh(
-				script: "cico node get -f value -c hostname -c comment --release=8 --retry-count=${cico_retries} --retry-interval=${cico_retry_interval}",
+				script: "cico node get -f value -c hostname -c comment --release=8 \
```

Comment:
I never tried breaking lines in Groovy, not sure it will work. Would be nice to know, finally!

Reply:
Yes, it was tested and it is working.